shared/secrets_masker/src/airflow_shared/secrets_masker/secrets_masker.py

```diff
@@ -70,38 +70,13 @@ def to_dict(self) -> dict[str, Any]: ...
 """Common terms that should be excluded from masking in both production and tests"""
 
 
-@cache
-def get_min_secret_length() -> int:
-    """Get minimum length for a secret to be considered for masking from airflow.cfg."""
-    from airflow.configuration import conf
-
-    return conf.getint("logging", "min_length_masked_secret", fallback=5)
-
-
-@cache
-def get_sensitive_variables_fields():
-    """Get comma-separated sensitive Variable Fields from airflow.cfg."""
-    from airflow.configuration import conf
-
-    sensitive_fields = DEFAULT_SENSITIVE_FIELDS.copy()
-    sensitive_variable_fields = conf.get("core", "sensitive_var_conn_names")
-    if sensitive_variable_fields:
-        sensitive_fields |= frozenset({field.strip() for field in sensitive_variable_fields.split(",")})
-    return sensitive_fields
-
-
 def should_hide_value_for_key(name):
     """
     Return if the value for this given name should be hidden.
 
     Name might be a Variable name, or key in conn.extra_dejson, for example.
     """
-    from airflow import settings
-
-    if isinstance(name, str) and settings.HIDE_SENSITIVE_VAR_CONN_FIELDS:
-        name = name.strip().lower()
-        return any(s in name for s in get_sensitive_variables_fields())
-    return False
+    return _secrets_masker().should_hide_value_for_key(name)
```

Comment:
Also kept the top level function, in case its being used in the wild. Undecided whether to keep or chop tbh